SRC/clarf1l.f

```diff
@@ -0,0 +1,265 @@
+*> \brief \b CLARF1L applies an elementary reflector to a general rectangular
+*              matrix assuming v(lastv) = 1, where lastv is the last non-zero
+*
+*  =========== DOCUMENTATION ===========
+*
+* Online html documentation available at
+*            http://www.netlib.org/lapack/explore-html/
+*
+*> \htmlonly
+*> Download CLARF1L + dependencies
+*> <a href="http://www.netlib.org/cgi-bin/netlibfiles.tgz?format=tgz&filename=/lapack/lapack_routine/clarf.f">
```

Comment:
Hi @EduardFedorenkov and @jprhyne, Can you change this link to clarf1l.f instead of clarf.f? The link won't work, but, next time, we release the link TGZ, ZIP and TXT will be created and the link will work. So we need to have the correct URLs there. And then ditto for each new files. So 1L x 1F x S x C for @EduardFedorenkov and 1L x 1F x D x Z for @jprhyne. Cheers, @langou

Reply:
done for {S, C}